packages/oscal-react-library/src/components/OSCALControl.tsx

```diff
@@ -1,32 +1,58 @@
 import React, { useEffect } from "react";
 import { styled } from "@mui/material/styles";
-import Card from "@mui/material/Card";
+import Card, { CardProps } from "@mui/material/Card";
 import CardContent from "@mui/material/CardContent";
 import Typography from "@mui/material/Typography";
 import Grid from "@mui/material/Grid";
 import OSCALControlLabel from "./OSCALControlLabel";
 import OSCALControlPart from "./OSCALControlPart";
 import OSCALControlModification from "./OSCALControlModification";
-import { OSCALAnchorLinkHeader } from "./OSCALAnchorLinkHeader";
+import { AnchorLinkProps, OSCALAnchorLinkHeader } from "./OSCALAnchorLinkHeader";
 import isWithdrawn from "./oscal-utils/OSCALCatalogUtils";
 import { propWithName } from "./oscal-utils/OSCALPropUtils";
 import { appendToFragmentPrefix } from "./oscal-utils/OSCALLinkUtils";
+import {
+  Control,
+  ImplementedRequirementElement,
+  SetParameterValue,
+  Alteration,
+} from "@easydynamics/oscal-types";
+import { EditableFieldProps } from "./OSCALEditableTextField";
 
-const OSCALControlCard = styled(Card, {
-  // https://github.com/mui/material-ui/blob/c34935814b81870ca325099cdf41a1025a85d4b5/packages/mui-system/src/createStyled.js#L56
-  shouldForwardProp: (prop) =>
-    !["childLevel", "withdrawn", "ownerState", "theme", "sx", "as"].includes(prop),
-})`
-  margin-top: 1em;
-  margin-bottom: 1em;
-  margin-left: ${(props) => (props.childLevel > 0 ? "1.5em" : "0")};
-  margin-right: ${(props) => (props.childLevel > 0 ? "1.5em" : "0")};
-  ${(props) => props.childLevel > 0 && "background-color: #fffcf0;"}
-  ${(props) =>
-    props.withdrawn && `text-decoration: line-through; color: ${props.theme.palette.grey[400]};`}
-`;
```

Comment:
Removing this has caused _more_ complaints about `withdrawn` and `childLevel` and the DOM and all that in the browser console. `shouldForwardProp` probably needs to be added back?